src/modules/user-conditions/UserConditions.module.scss

```diff
@@ -3,6 +3,9 @@
 @use "src/styles/abstracts" as *;
 
 $terms-of-service-background-color: #ebfbf8;
+$top-navigation-bar-height: 10rem;
+$header-and-buttons-height: 31.4rem;
+$header-and-buttons-height-mobile: 21.8rem;
```

Comment:
is there a way to use one of the $spacer variables for this? Just to make things more consistent.

Reply:
i'll replace it with a grid layout that doesn't require these calc constants in a future pr